internal/config/cgroupmanager/cgroupfsmanager_unsupported.go

```diff
@@ -0,0 +1,17 @@
+// +build !linux
+
+package cgroupmanager
+
+import (
+	"fmt"
+)
+
+type CgroupManager interface {
+	GetContainerCgroupPath(string, string) string
+	// cgroup parent, cgroup path, error
+	GetSandboxCgroupPath(string, string) string
+}
+
+func InitializeCgroupManager(cgroupManager string) (CgroupManager, error) {
+	return nil, fmt.Errorf("not implemented yet")
```

Comment:
nit: `s/fmt.Errorf/errors.New/` since you do not using any `%`-style formatting

Reply:
fixed!